### DISCUSSION TITLE
Unsupported Order: 257

### DISCUSSION THREAD

**Discussion by japresa** [idaholab/moose#19207] [Q&A General] [answered]
Hello. I'm running SAM/MOOSE simulation for a reactor geometry I made using TRELIS and am confused on why I'm getting the following error. Couldn't understand the error-code or find any information online where to start debugging.

 The error occurs after adding component physics. Here is a screen shot of some error code.

Anything helps!

![image](https://user-images.githubusercontent.com/79496775/138823252-47be75f5-61b2-4e74-9a16-f7702255a8e6.png)

**Comment by ABallisat** [comment, marked as answer]
I normally get this when the variables are of a higher order than the elements in your mesh, e.g. using a second order variable with quad4 elements. What elements and order of variables are you using?

**Reply by japresa** [reply]
Thank you so much! Just realized I was using second order when trying element HEX8. I changed all my second order equations to first order and was able to get the simulation running.
